snippets/go.json

```diff
@@ -254,6 +254,16 @@
 			"prefix": "sort",
 			"body": "type ${1:SortBy} []${2:Type}\n\nfunc (a $1) Len() int           { return len(a) }\nfunc (a $1) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }\nfunc (a $1) Less(i, j int) bool { ${3:return a[i] < a[j]} }",
 			"description": "Snippet for a custom sort.Sort interface implementation, for a given slice type."
+		},
+		"test got want error": {
+			"prefix": "gwe",
+			"body": "t.Errorf(\"${1:funcName}(${2:args}) = ${3:got}, want ${4:want}\",${5:vars})",
```

Comment:
I think this looks good but should we start with something simpler along the line of `t.Errorf("Expected %d, got %d", expected, actual)`?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

My question is whether it is general enough for all go extension users would find useful and worth being included in this extension rather than individual user's snippet. I used and suggested this test error format many times during go readability review, but i also observed many cases where other formats are preferred.